cmd/mta.go

```diff
@@ -21,10 +21,16 @@ var existCmdName string
 var existCmdPath string
 var getBuildParametersCmdPath string
 var getBuildParametersCmdExtensions []string
+var getGlobalParametersCmdPath string
+var getGlobalParametersCmdExtensions []string
 var updateBuildParametersCmdPath string
 var updateBuildParametersCmdData string
 var updateBuildParametersCmdForce bool
 var updateBuildParametersCmdHashcode int
+var updateGlobalParametersCmdPath string
+var updateGlobalParametersCmdData string
+var updateGlobalParametersCmdForce bool
```

Comment:
I don't think we should support force-updating the build parameters because there is a way to retrieve them.
Force-adding a module or resource will not delete existing modules or resources so it isn't bad to support it (at worst it will add a module or resource that already exists and make the mta invalid), but force-updating parameters can delete the existing parameters. It's the same for build parameters but unfortunately we already support it so it's a problem to remove it now (and I guess we added it because there was no way to get the build parameters previously so the existing build parameters were deleted anyway).
Please remove this variable and the support for it everywhere relevant.